Models/TreeModel.cpp

```diff
@@ -689,13 +689,13 @@ QModelIndex TreeModel::addNode(const Message &child, const QModelIndex &parent)
   return insert(insertParent, pos, child);
 }
 
-QModelIndex TreeModel::duplicateNode(const QModelIndex & /*index*/) {
-  /*Node *node = IndexToNode(index);
-  if (node->repeated_model) {
-    return node->mapFromSource(node->repeated_model->duplicate(mapToSource(index)));
-  }*/
-  qDebug() << rowCount() << " rows";
-  return QModelIndex();
+QModelIndex TreeModel::duplicateNode(const QModelIndex & index) {
+  Node *node = IndexToNode(index);
+  if (node->IsRepeated()) {
```

Comment:
Like this change. But please just leave the if() commented if it's not being used.